cli/command/service/ps.go

```diff
@@ -32,6 +33,7 @@ func newPsCommand(dockerCli *command.DockerCli) *cobra.Command {
 		},
 	}
 	flags := cmd.Flags()
+	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Only display IDs")
```

Comment:
Looks good, wondering if we should use `Only display task IDs` (there's so many ID's related to this overview, it became a bit confusing to me)

@vdemeester wdyt?